src/Core/MainLoop.js

```diff
@@ -153,6 +168,16 @@ MainLoop.prototype._update = function _update(view, updateSources, dt) {
             view.execFrameRequesters(MAIN_LOOP_EVENTS.AFTER_LAYER_UPDATE, dt, this._updateLoopRestarted, geometryLayer);
         }
     }
+
+    if (context.distance.min != Infinity) {
+        // Multiply min/max by (0.9, 1.1) to be on the safe side
+        view.camera.camera3D.near = Math.max(0.1, 0.9 * context.distance.min);
```

Comment:
As the `min` distance is calculated relative to the camera's position, close objects may be truncated by the near.
This min distance must be projected on the z-axis of the camera

Reply:
I don't understand, can you rephrase?

Reply:
`view.camera.camera3D.near ` it 's the distance between the near plane and the camera's position, on camera's  z Axis.

`context.distance.min` is the distance between camera's position and bbox. This distance should be reprojected on camera's  z Axis.

Reply:
`min` is the minimum distance between the camera's center and any (visible) objects in the scene.
So I don't see why / what should be reprojected on camera's z axis.

Can you make a drawing of a case where using the `min` distance as the `near` is incorrect?